scripts/sources.conf

```diff
@@ -18,6 +18,7 @@ export REDIS_CACHE_HOST=localhost
 export REDIS_CACHE_PORT=6379
 export SOURCES_API_HOST=http://localhost
 export SOURCES_API_PORT=$PORT
+export ARN_ROLE=arn:aws:iam::123456789:role/redhat-provisioning-1
```

Comment:
This needs to be defined by everyone separately as @adiabramovitch has wrote already :thinking:
Shall we keep blank/commented out and fail seeding if left empty (with message what to do)?

Reply:
Yeah I was just lazy.